internal/types/types.go

```diff
@@ -1700,3 +1701,26 @@ type Webhook struct {
 	CreatedByUserID int32
 	UpdatedByUserID int32
 }
+
+// CodeHostURN is an identifier for a unique code host in the form
+// of its host URL.
+type CodeHostURN struct {
+	urlString string
```

Comment:
We do have a type system :)

```suggestion
	url string
```